docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -162,6 +162,8 @@ You can create an XML file to configure a `CacheManager`.
 ----
 include::{sourcedir310}/ehcache-xml/src/test/resources/configs/docs/getting-started.xml[tags=gettingStarted]
 ----
+NOTE: The use of memory units (such as 'B', 'kB' or 'MB') for heap resources is deprecated and will be removed in a future version.This only applies to heap tiers.
```

Comment:
This sample doesn't even reference byte-based units. I think we can safely drop this note.

Reply:
Conversely there is a link to byte-based heap documentation at the end of this file that we should probably remove. Don't want to lead more people to using a deprecated feature (around line 260ish)

Reply:
There's also a reference to byte based sizing in the `performance.adoc` file that should probably get a warning/note.

Reply:
I have deleted this, I had put it for precautionary measure.

Reply:
Done, removed the entire block

Reply:
Added a warning note there as well (at the beginning) , I have changed all my prev NOTE to WARNING